env/release.mk

```diff
@@ -17,14 +17,14 @@ export RPMDIR=/srv/releases/jenkins/redhat${RELEASELINE}
 export SUSEDIR=/srv/releases/jenkins/opensuse${RELEASELINE}
 
 # where to put repository index and other web contents
-export  RPM_WEBDIR=/var/www/pkg.jenkins-ci.org.staging/redhat${RELEASELINE}
-export SUSE_WEBDIR=/var/www/pkg.jenkins-ci.org.staging/opensuse${RELEASELINE}
-export  DEB_WEBDIR=/var/www/pkg.jenkins-ci.org.staging/debian${RELEASELINE}
+export  RPM_WEBDIR=/var/www/pkg.jenkins.io.staging/redhat${RELEASELINE}
```

Comment:
This part of the changes will break things unless accompanied by a change to [this script|https://github.com/jenkins-infra/jenkins-infra/blob/staging/dist/profile/files/mirrorbrain/sync.sh#L64]